<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-998S31Q" ignored="0" name="verifying Clients Window using chrome" tests="8" failures="1" timestamp="23 Jun 2021 16:12:59 GMT" time="220.054" errors="0">
  <testcase name="executeVerifyClientTitle" time="7.995" classname="ExecuteClasses.ExecuteClient"/>
  <testcase name="executeVerifyClientsIdsearchBarText" time="6.783" classname="ExecuteClasses.ExecuteClient"/>
  <testcase name="executeVerifyClientsNameSearchBarText" time="7.761" classname="ExecuteClasses.ExecuteClient"/>
  <testcase name="executeVerifyClientsWindowVisibility" time="11.318" classname="ExecuteClasses.ExecuteClient"/>
  <testcase name="executeVerifySearchingFunctionality" time="8.02" classname="ExecuteClasses.ExecuteClient"/>
  <testcase name="executeVerifySearchingFunctionality" time="8.298" classname="ExecuteClasses.ExecuteClient"/>
  <testcase name="executeVerifySearchingFunctionality" time="9.71" classname="ExecuteClasses.ExecuteClient"/>
  <testcase name="executeVerifyclientsSearchName" time="12.247" classname="ExecuteClasses.ExecuteClient">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [.] but found [Claydon]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [.] but found [Claydon]
at ExecuteClasses.ExecuteClient.executeVerifyclientsSearchName(ExecuteClient.java:96)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- executeVerifyclientsSearchName -->
</testsuite> <!-- verifying Clients Window using chrome -->
